src/Loaders/LoadingSpinner.tsx

```diff
@@ -0,0 +1,76 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx, keyframes } from "@emotion/core";
+import { colors } from "../colors";
+
+export type Size = "large" | "medium" | "small" | "tiny" | "icon";
```

Comment:
Sure! Im not quite clear what specifically you're suggesting though, `"large" | "medium" | "small" | "extraSmall" | "2extraSmall"`? To me it feels weird to say '2' with the full word, but shortening to `2xs` is weird when using full names for all other component sizes. One thing to note is `icon` might make it clearer which size matches the icon size for buttons etc, instead of having to try out multiple sizes

Reply:
@justinanastos I just got to @daniman s comment below about the timing changing when used as an icon. What would you think about removing `icon` as a size and adding it as its own flag? Im a bit conflicted on having multiple props affect size but if we do want to have the different animation at that size it might make sense (We can also make the type so only one of `icon` or `size` could be passed).

Reply:
I think `2xs` stands for `extraExtraSmall` and is more readable than `xxs`, especially as things get more `extra` 😆. I'm into keeping the names expanded, since users can add their own semantics on top of these if they want. `2extraSmall` makes sense to me. I think the animation changes depending on the size are a space-kit concern. Their use is an application concern, so I think `icon` should not show up in the api

Reply:
This suggestion is coming from this comment by @timbotnik :

https://github.com/mdg-private/engine-frontend/pull/1606#pullrequestreview-277500719

> This will be great, however I would prefer if we kept parallel with the Tailwind text naming conventions? For example, instead of `sk-text-small` we should use `sk-text-sm`, instead of `sk-text-xxlarge` we should use `sk-text-2xl`, etc.
> 
> > .text-xs | font-size: .75rem;
> > .text-sm | font-size: .875rem;
> > .text-base | font-size: 1rem;
> > .text-lg | font-size: 1.125rem;
> > .text-xl | font-size: 1.25rem;
> > .text-2xl | font-size: 1.5rem;
> > .text-3xl | font-size: 1.875rem;
> > .text-4xl | font-size: 2.25rem;
> > .text-5xl | font-size: 3rem;
> > .text-6xl | font-size: 4rem;

We've been using `xs`, `sm`, `base`, `lg`, `xl`, `2xl`, `3xl`, etc for a year and the language feels right to me. This is subjective, and we're talking about bringing something I personally feel very comfortable with in graph manager to Space Kit. I want to point out that this is *not* for the sake of consistency; this is for the sake of I think this is a good API.

Reply:
🤔 I'm comfortable with either as long as the names are consistent within space kit, ie

https://github.com/apollographql/space-kit/blob/16a47a1017b4d5c512e3e56406331993ef0309cb/src/Button/Button.tsx#L172

I lean towards full english for space-kit, since the sizes stand on their own. I've found short names with tailwinds are useful and necessary, since we write so many of them. Because they are constant throughout our application the language/effects are top of mind with little recall effort and brevity means efficiency. I believe that these components will be used once or twice per component/file, so brevity is less important and expressiveness is more. That's my opinion. I'll be happy either way and defer to your decision. Look forward to learning/seeing it used 🎉

Reply:
I can handle that @evans . I do not want to have ambiguity though:

```jsx
// I don't know if `tiny` or `icon` is smaller. 
"large" | "medium" | "small" | "tiny" | "icon";
```

How about this:
```jsx
"large" | "medium" | "small" | "xsmall" | "2xsmall";
```

Reply:
I can handle that @evans . I do not want to have ambiguity though:

```jsx
// I don't know if `tiny` or `icon` is smaller. 
"large" | "medium" | "small" | "tiny" | "icon";
```

How about this:
```jsx
"large" | "medium" | "small" | "xsmall" | "2xsmall";
```